Userland/Libraries/LibWeb/HighResolutionTime/Performance.cpp

```diff
@@ -39,8 +39,7 @@ JS::GCPtr<NavigationTiming::PerformanceTiming> Performance::timing()
 
 double Performance::time_origin() const
 {
-    auto origin = m_timer.origin_time();
-    return (origin.tv_sec * 1000.0) + (origin.tv_usec / 1000.0);
+    return static_cast<double>(m_timer.origin_time().to_milliseconds());
```

Comment:
This technically cuts off some precision from the timestamp. @linusg  do we care?

Reply:
Nope, that's ok - no one can expect to get sub-millisecond precision on the web (which I think is what you're referring to).